packages/kit/src/types.d.ts

```diff
@@ -137,8 +143,8 @@ export type RenderOptions = {
 	entry?: string;
 	root?: SSRComponent['default'];
 	setup?: {
-		prepare?: ({
-			headers: Headers
+		prepare?: (ctx: {
```

Comment:
`ctx` is possibly a bit vague given that we're bandying the word `context` around in a couple of places — maybe something like this?

```suggestion
		prepare?: (incoming: {
```